docs/ref/forms/api.txt

```diff
@@ -1448,14 +1477,13 @@ If you need to access some additional information about a form field in a
 template and using a subclass of :class:`~django.forms.Field` isn't
 sufficient, consider also customizing :class:`~django.forms.BoundField`.
```

Comment:
I mean something like this:
```diff
 Customizing ``BoundField``
 ==========================
 
-If you need to access some additional information about a form field in a
-template and using a subclass of :class:`~django.forms.Field` isn't
-sufficient, consider also customizing :class:`~django.forms.BoundField`.
-
 A form can override :attr:`.Form.bound_field_class` and a form field can take
 an optional :attr:`.Field.bound_field_class` constructor argument.
 
+For example, you may need to access some additional information about a form
+field in a template and using a subclass of :class:`~django.forms.Field` isn't
+sufficient.
+
 If you have a ``GPSCoordinatesField``, for example, and want to be able to
 access additional information about the coordinates in a template, this could
 be implemented as follows::
@@ -1500,6 +1500,29 @@ be implemented as follows::
 Now you can access the country in a template with
 ``{{ form.coordinates.country }}``.
 
+You may also want to customize the default form field template rendering. For
+example, you can override :meth:`.BoundField.label_tag` to add a custom class::
+
+
+   class StyledLabelBoundField(BoundField):
+       def label_tag(self, contents=None, attrs=None, label_suffix=None, tag=None):
+           attrs = attrs or {}
+           attrs["class"] = "wide"
+           return super().label_tag(contents, attrs, label_suffix, tag)
+
+
+   class UserForm(forms.Form):
+       bound_field_class = StyledLabelBoundField
+       name = CharField()
+
+This would update the default form rendering:
+
+.. code-block:: pycon
+
+   >>> f = UserForm()
+   >>> print(f["name"].label_tag)
+   <label for="id_name" class="wide">Name:</label>
+
 A project can define a project-wise ``BoundField`` class using
 :attr:`.BaseRenderer.bound_field_class` and :setting:`FORM_RENDERER`.
```